slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/interchange/InterchangeV5Importer.java

```diff
@@ -107,13 +112,19 @@ public void importData(final InputStream input) throws IOException {
     }
   }
 
-  private void parseValidator(final Handle handle, final JsonNode node)
+  private void parseValidator(
+      final Handle handle, final JsonNode node, final Optional<List<String>> pubkeys)
       throws JsonProcessingException {
     if (node.isArray()) {
       throw new IllegalStateException("Element of 'data' was not an object");
     }
     final ObjectNode parentNode = (ObjectNode) node;
     final String pubKey = parentNode.required("pubkey").textValue();
+
+    if (pubkeys.isPresent() && !pubkeys.get().contains(pubKey)) {
```

Comment:
Can you add a test for this case

Reply:
done - added one more entry to the slashing data - the `importLoadsSlashingData` validates that only 1 validator entry gets added to the DB